naples/models/user.py

```diff
@@ -88,3 +92,12 @@ def __repr__(self):
     def json(self):
         u = s.User.model_validate(self)
         return u.model_dump_json()
+
+    @classmethod
+    def get_user_by_email(cls, email: str, session: orm.Session | None = None) -> Self | None:
```

Comment:
I think, session should not be optional argument here. Caller of this method should ensure that db session exists